internal/commands/base/base.go

```diff
@@ -230,6 +231,10 @@ func BuildCommands(rootCmd *cobra.Command, conf *config.Config) {
 	commands.BuildCommand(partneraccount.CreateCommand(), partnerAccountCommand.Cobra(), conf)
 	commands.BuildCommand(partneraccount.ListCommand(), partnerAccountCommand.Cobra(), conf)
 
+	// Audit log operations
+	auditlogCommand := commands.BuildCommand(auditlog.BaseAuditLogCommand(), rootCmd, conf)
```

Comment:
Could also have this under account command similarly than we have audit-logs page under account details in hub, but no strong opinions on this from my side 🤔